src/BizHawk.Emulation.Cores/Computers/AmstradCPC/Hardware/Display/CRCT_6845.cs

```diff
@@ -868,7 +868,7 @@ private bool ReadRegister(ref int data)
 				case 9:
 				case 10:
 				case 11:
-					if ((int)ChipType == 0 || (int)ChipType == 1)
+					if (ChipType == 0 || (int)ChipType == 1)
```

Comment:
I think I would prefer ChipType == CRCTType.HD6845S || ChipType == CRCTType.HD6845S || ChipType == CRCTType.UM6845R here and in the other cases, the whole point of the enum would be able to say if it's this chip or that chip by name

Reply:
I think the last comparison may be typoed now, it was comparing for bits 0 or 2 and it's now effectively against 0 1 or 2